src/includes/set-release/react-native.mdx

```diff
@@ -0,0 +1,14 @@
+You will also need to past a distribution to `dist` for events to correctly be attributed to a release. Usually this value is the build number, for example `52`. This value only needs to be unique to each release.
+
+```javascript
+Sentry.init({
+  release: "my-project-name@2.3.12",
+  dist: "52",
+});
+```
+
+<Alert level="info" title="Source Maps">
+
+When you set the `release` and `dist` in a React Native app, the automatic source maps upload script will no longer work, you will have to [manually upload sourcemaps](/platforms/react-native/sourcemaps).
```

Comment:
```suggestion
When you set a custom `release` and `dist` in a React Native app, the automatic source maps upload script will no longer work because it does not detect custom values. You will need to [manually upload source maps](/platforms/react-native/sourcemaps).
```